src/operator/contrib/nnvm_to_onnx.cc

```diff
@@ -248,8 +259,12 @@ void ConvertPooling(NodeProto* node_proto, const NodeAttrs& attrs,
   AttributeProto* const pads = node_proto->add_attribute();
   pads->set_name("pads");
   pads->set_type(AttributeProto::INTS);
-  for (int kval : pad) {
-    pads->add_ints(static_cast<int64>(kval));
+
+  // Convert from MXNet symetric pads to ONNX non-symetric by running through padding twice.
+  for (int i =0; i < 2; i++) {
```

Comment:
This was already done by roshani if I remember it correctly. are we duplicating code here?

Form my understanding, we would only need onnx->Tensorrt since the mxnet onnx part should be handled by the existing solution. It feels like we are developing to parallel approaches here - but I could be mixing things up

Reply:
Yeah I want to have a conversation with them offline, I've already brought this up a few times.  The trick here is we need to export to ONNX from native code.  This is more or less 100% duplication (minus tests).

Reply:
Agree. Shall we wait with all that until after 1.4 or merge this PR to upgrade Tensorrt and do the refactor afterwards?

Reply:
Wait until after 1.4 for sure, it's a big change that will require a lot of discussion.

Reply:
Spoke to Roshani about this and we agreed that we'd keep both implementation for both for the time being.  We think the support level is better in python, so that will be the main place to do import / exports from python.  This implementation will try to duplicate their export logic, but it will only include the operators that TRT can make use of.